### ISSUE TITLE
Microsoft Connect Has Been Retired

### ISSUE THREAD

**Issue by Mottie** [Fyrd/caniuse#5870]
I was looking at https://caniuse.com/flexbox today. In the known issues tab are several links to bugs in Microsoft Connect. Clicking on them redirects me to https://docs.microsoft.com/en-us/collaborate/connect-redirect which states that Microsoft Connect has been retired.

**Comment by Schweinepriester** [comment]
I started looking into this and there appear to be [20 links](https://github.com/Fyrd/caniuse/search?q=connect.microsoft.com) left:

<img width="635" alt="Bildschirmfoto 2021-05-02 um 02 10 25" src="https://user-images.githubusercontent.com/2644614/116797989-a0fe7800-aaeb-11eb-94b3-e4c6d894a619.png">

Started searching and found #4067. Looks like that was not complete.

So I wanted to try to replace them with ones from <https://web.archive.org/>, but already the first one I tried doesn't have a real entry:
- https://connect.microsoft.com/IE/feedback/details/816293/ie11-flexbox-with-min-height-not-vertically-aligning-with-align-items-center
- https://web.archive.org/web/2018*/https://connect.microsoft.com/IE/feedback/details/816293/ie11-flexbox-with-min-height-not-vertically-aligning-with-align-items-center
- Google neither: https://webcache.googleusercontent.com/search?q=cache%3Ahttps%3A%2F%2Fconnect.microsoft.com%2FIE%2Ffeedback%2Fdetails%2F816293%2Fie11-flexbox-with-min-height-not-vertically-aligning-with-align-items-center&ei=AuqNYN3xOPT87_UPqaWn4A4

At least some appear to be in the archive. So I'll replace those.

Whats worse though, Edge issues appear to be redirected as well, for example <https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/250661/> => <https://developer.microsoft.com/en-us/microsoft-edge/>.

And of `developer.microsoft.com` there are [even more links, 48](https://github.com/Fyrd/caniuse/search?q=developer.microsoft.com):

<img width="624" alt="Bildschirmfoto 2021-05-02 um 02 10 38" src="https://user-images.githubusercontent.com/2644614/116797995-aa87e000-aaeb-11eb-88a7-6557e5a9f097.png">

But as we don't know if there even is an entry in the archive, we can't just regex replace them, but have to check each one…

---

While I get to work, per https://github.com/Fyrd/caniuse/issues/4067#issuecomment-364565039 calling @dstorey & @molant: Is the content of <https://developer.microsoft.com/en-us/microsoft-edge/platform/issues> still online somewhere?

**Comment by Schweinepriester** [comment]
connect.microsoft.com links not available anywhere (including without the string after the id):
- https://connect.microsoft.com/IE/feedback/details/1847897/globalalpha-ignored-when-drawing-svg-to-canvas
- http://connect.microsoft.com/IE/feedback/details/762719/css3-calc-bug-inside-transition-or-transform
- https://connect.microsoft.com/IE/feedbackdetail/view/920928/ie-11-css-transition-property-not-working-for-svg-elements
- https://connect.microsoft.com/IE/feedback/details/816293/ie11-flexbox-with-min-height-not-vertically-aligning-with-align-items-center
- https://connect.microsoft.com/IE/feedback/details/829392/calling-getboundingclientrect-on-an-html-element-that-has-not-been-added-to-the-dom-causes-unspecified-error
- https://connect.microsoft.com/IE/feedback/details/885747/ie-11-fires-the-input-event-when-a-input-field-with-placeholder-is-focused
- https://connect.microsoft.com/IE/feedbackdetail/view/800076/internet-explorer-11-lost-support-for-non-rgb-a-png-format-images-inside-favicon-ico
- https://connect.microsoft.com/IE/feedback/details/817132/ie-11-childnodes-are-missing-from-mutationobserver-mutations-removednodes-after-setting-innerhtml

**Comment by Schweinepriester** [comment]
developer.microsoft.com links not available in the web archive:
- https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/104773/
- https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/18676405/
- https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/18675657/
- https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/16765952/
- https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/11495448/
- https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/11676012/
- https://developer.microsoft.com/microsoft-edge/platform/issues/1328019/
- https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/20066595/
- https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/101525/ (3x in the JSONs)
- https://developer.microsoft.com/microsoft-edge/platform/issues/4660045/
- https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/6274479/
- https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/4365703/

**Comment by Schweinepriester** [comment]
Now that #5876 has been merged, I don't know what else we can do. Simply removing the broken links isn't useful either…

**Comment by molant** [comment]
>  Is the content of https://developer.microsoft.com/en-us/microsoft-edge/platform/issues still online somewhere?


I don't think it is. The issue tracker got removed shortly after I left the team 😔
